robyn/openapi.py

```diff
@@ -236,9 +236,9 @@ def get_path_obj(
         name: str,
         description: str,
         tags: List[str],
-        query_params: Optional[TypedDict],
-        request_body: Optional[TypedDict],
-        return_annotation: Optional[TypedDict],
+        query_params: Optional[Dict],
+        request_body: Optional[Dict],
+        return_annotation: Optional[Dict],
```

Comment:
What error prompted these changes??

Reply:
I'll check tomorrow.
I think the solution (to follow all the docs I can find) would be to have our own class of TypedDict rather than use TypedDict directly.